packages/connect-ui/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@trezor/connect-ui",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "main": "src/index",
+    "scripts": {
+        "lint": "eslint '**/*.{ts,tsx,js}'",
+        "test:unit": "jest -c ../../jest.config.base.js --passWithNoTests",
+        "type-check": "tsc --build"
+    },
+    "dependencies": {
+        "@trezor/components": "*",
+        "@trezor/urls": "*",
+        "react": "17.0.2",
+        "styled-components": "^5.3.5"
+    }
```

Comment:
Probably yes. I must say that I would greatly appreciate some kind of automation here. But it is probably not trivial to achieve since we want to list some libs only in root and others should be listed inside packages package.json :thinking:

Reply:
af70447c44e745d586cdc4b25b6025d541fdf0a2